botorch/optim/parameter_constraints.py

```diff
@@ -472,27 +535,26 @@ def make_scipy_nonlinear_inequality_constraints(
         values and Jacobians and a string indicating the associated constraint
         type ("eq", "ineq"), as expected by `scipy.minimize`.
     """
-    if not isinstance(nonlinear_inequality_constraints, list):
-        raise ValueError(
-            "`nonlinear_inequality_constraints` must be a list of callables, "
-            f"got {type(nonlinear_inequality_constraints)}."
-        )
 
     scipy_nonlinear_inequality_constraints = []
-    for nlc in nonlinear_inequality_constraints:
-        if _arrayify(nlc(x0)).item() < NLC_TOL:
+    for constraint in nonlinear_inequality_constraints:
+        if not isinstance(constraint, tuple):
+            raise ValueError(
+                f"A nonlinear constraint has to be a tuple, got {type(constraint)}."
+            )
+        if len(constraint) != 2:
+            raise ValueError(
+                "A nonlinear constraint has to be a tuple of length 2, "
+                f"got length {len(constraint)}."
+            )
+        nlc, is_intra = constraint
+        if not nonlinear_constraint_is_fulfilled(nlc, is_intra, x0.reshape(shapeX)):
```

Comment:
just some formatting nits; let's call most functions with kwargs (except where things are very obvious)
```suggestion
        nlc, is_intrapoint = constraint
        if not nonlinear_constraint_is_fulfilled(nlc, is_intrapoint=is_intrapoint, x= x0.reshape(shapeX)):
```